version.txt

```diff
@@ -1 +1 @@
-1.3.4.376
```

Comment:
the version.txt file is used for update checks, so please remove this change.
I'll update that file when I make a new release.

Reply:
Change removed in new commit